packages/components/components/elvis-accordion/src/react/elvia-accordion.spec.tsx

```diff
@@ -4,20 +4,109 @@ import React from 'react';
+            consequatur officiis animi nostrum facilis tenetur temporibus alias aliquam eum praesentium nisi?
+            Repudiandae dolor perspiciatis est sapiente porro. Lorem ipsum dolor sit amet consectetur
+            adipisicing elit. Voluptatibus ipsam, deleniti enim consequatur officiis animi nostrum facilis
+            tenetur temporibus alias aliquam eum praesentium nisi? Repudiandae dolor perspiciatis est sapiente
+            porro.
+          </div>
+        }
+      />
+      <Accordion
+        size="large"
+        openLabel="Large open accordion"
+        closeLabel="Large open accordion"
+        isOpen
+        content={
+          <div>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatibus ipsam, deleniti enim
+            consequatur officiis animi nostrum facilis tenetur temporibus alias aliquam eum praesentium nisi?
+            Repudiandae dolor perspiciatis est sapiente porro. Lorem ipsum dolor sit amet consectetur
+            adipisicing elit. Voluptatibus ipsam, deleniti enim consequatur officiis animi nostrum facilis
+            tenetur temporibus alias aliquam eum praesentium nisi? Repudiandae dolor perspiciatis est sapiente
+            porro.
+          </div>
+        }
+      />
+      <Accordion
+        openLabel="Overflow accordion"
+        closeLabel="Overflow accordion"
+        type="overflow"
+        content={
+          <div>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatibus ipsam, deleniti enim
+            consequatur officiis animi nostrum facilis tenetur temporibus alias aliquam eum praesentium nisi?
+            Repudiandae dolor perspiciatis est sapiente porro. Lorem ipsum dolor sit amet consectetur
+            adipisicing elit. Voluptatibus ipsam, deleniti enim consequatur officiis animi nostrum facilis
+            tenetur temporibus alias aliquam eum praesentium nisi? Repudiandae dolor perspiciatis est sapiente
+            porro.
+          </div>
+        }
+      />
+      <Accordion
+        openLabel="Click to open accordion"
+        closeLabel="Click to open accordion"
+        content={
+          <div>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatibus ipsam, deleniti enim
+            consequatur officiis animi nostrum facilis tenetur temporibus alias aliquam eum praesentium nisi?
+            Repudiandae dolor perspiciatis est sapiente porro. Lorem ipsum dolor sit amet consectetur
+            adipisicing elit. Voluptatibus ipsam, deleniti enim consequatur officiis animi nostrum facilis
+            tenetur temporibus alias aliquam eum praesentium nisi? Repudiandae dolor perspiciatis est sapiente
+            porro.
+          </div>
+        }
+      />
+    </div>,
+  );
+  await page.getByRole('button', { name: 'Click to open accordion', exact: true }).click();
+  await page.waitForTimeout(300);
+  await percySnapshot(page, 'Accordions', { widths: [900] });
```

Comment:
There is no point in running these tests for multiple widths as the component doesn't change for desktop vs. mobile, so I'm here explicitly only running it for one screen width to reduce the number of Percy screenshots

Reply:
Does the typography automatically change between mobile and desktop?

Reply:
Generally no, however a user can override the typography used for the buttons and that could give it some changes. 
However, I would argue that such a minor change would fall outside the scope of these tests, and should instead be covered by typography-specific tests (which I believe we have in the Elvis percy-tests).